e2e/node/basic/callAndPoll.test.ts

```diff
@@ -0,0 +1,17 @@
+import { HttpAgent, fromHex, callAndPoll } from '@dfinity/agent';
+import { expect, describe, it, vi } from 'vitest';
+describe('call and poll', () => {
+  it('should handle call and poll', async () => {
+    vi.useRealTimers();
+
+    const options = {
+      canisterId: 'tnnnb-2yaaa-aaaab-qaiiq-cai',
+      methodName: 'inc_read',
+      agent: await HttpAgent.create({ host: 'https://icp-api.io' }),
+      arg: fromHex('4449444c0000'),
+    };
+
+    const certificate = await callAndPoll(options);
+    expect(certificate instanceof ArrayBuffer).toBe(true);
```

Comment:
Are you going to enhance the tests? e.g. asserting that the certificate return is the expected value or asserting that the function correctly throws if there was an error?

Reply:
Yes, this should have been marked as a draft. I wanted feedback on the inputs and return values before proceeding. If you have real-world test cases this feature will be used for, those would be great to add as tests as well, beyond the basic cases!